### ISSUE TITLE
'list' object has no attribute 'status_code'

### ISSUE THREAD

**Issue by azuravian** [clinton-hall/nzbToMedia#1500]
I'm using nzbtomedia on a QNAP NAS and it's been working great for a while.  Today, I started having problems and made sure to clone the latest commit and I now get the following error from nzbToRadarr.

```
INFO | Fri Jan 04 2019 21:15:38 | nzbToRadarr: Exception AttributeError: "'NoneType' object has no attribute 'path'" in <bound method PosixProcess.__del__ of <core.utils.PosixProcess object at 0x7f25208f1810>> ignored
-- | -- | --
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: AttributeError: 'list' object has no attribute 'status_code'
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: if result.status_code == 0:
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: File "/share/CACHEDEV1_DATA/.qpkg/NZBGet/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 767, in process
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: failure_link=failure_link)
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: File "/share/CACHEDEV1_DATA/.qpkg/NZBGet/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 856, in main
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: result = nzbToMedia.main(sys.argv, section)
INFO | Fri Jan 04 2019 21:15:37 | nzbToRadarr: File "/share/CACHEDEV1_DATA/.qpkg/NZBGet/nzbget/scripts/nzbToMedia/nzbToRadarr.py", line 255, in <module>
```

**Comment by labrys** [comment]
Hotfix will be posted shortly.

**Comment by azuravian** [comment]
Ah, so it's not me.  I figured I must have done something wrong. ;)

**Comment by labrys** [comment]
@azuravian Any chance you can post a full debug log so I can do a more targeted hotfix?

**Comment by labrys** [comment]
Fixed in v12.0.2

**Comment by azuravian** [comment]
Thank you!
